test/python/transpiler/test_dense_layout.py

```diff
@@ -16,14 +16,15 @@
 
 import numpy as np
 
-from qiskit import QuantumRegister, QuantumCircuit
+from qiskit import QuantumRegister, QuantumCircuit, ClassicalRegister
 from qiskit.circuit import Parameter
 from qiskit.circuit.library import CXGate, UGate, ECRGate, RZGate
 from qiskit.transpiler import CouplingMap, Target, InstructionProperties, TranspilerError
 from qiskit.transpiler.passes import DenseLayout
 from qiskit.converters import circuit_to_dag
 from qiskit.test import QiskitTestCase
 from qiskit.providers.fake_provider import FakeTokyo
+from qiskit.visualization.gate_map import *
```

Comment:
This seems questionable?

Reply:
debug code. Thanks for spotting. removed in e390d476